books/RayTracingInOneWeekend.html

```diff
@@ -1456,8 +1455,8 @@
 
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ delete
-    using std::shared_ptr;
     using std::make_shared;
+    using std::shared_ptr;
```

Comment:
These don't need to be flipped. When introduced in `The hittable_list class` (Listing 21) 
they are ordered: 
```
using std::shared_ptr;
using std::make_shared;
```

Reply:
Hmm, some of these reorderings must have been dropped. Since adding a new using statement for fabs here, the random order looked more annoying, so I'll grow this in sorted order. Will update.